public/doc/swagger-2-v0.0.1.yaml

```diff
@@ -1121,6 +1121,27 @@ paths:
             "$ref": "#/definitions/Task"
         404:
           description: Not found
+    patch:
```

Comment:
I don't think we should be updating this version of the API any more.

Reply:
Good idea, on the contrary, should we be bumping the version by a patch level or something then for this change? So instead of modifying v0.1.0, should I be making a new v0.1.1 with the patch change?

Reply:
Not now, I think we need to figure out the deployment strategy first so that we can collect a set of changes and release that as a version.  For now I see v0.0 as deprecated and v0.1 as current.